lib/tests/modules/alias-with-priority-can-override.nix

```diff
@@ -32,7 +32,7 @@ with lib;
 
   imports = [
     # Create an alias for the "enable" option.
-    (mkAliasOptionModuleWithPriority [ "enableAlias" ] [ "enable" ])
+    (mkAliasOptionModule [ "enableAlias" ] [ "enable" ])
 
     # Disable the aliased option, but with a default (low) priority so it
```

Comment:
the comment says `with a default (low) priority`, but actual code is `enableAlias = lib.mkForce false;`

Reply:
@danbst Good catch!

I fixed this in commit dcbd136.